scripts/setup-test-data/README.md

```
@@ -0,0 +1,86 @@
+The script uses the `GITHUB_TOKEN` environment variable for authenticated requests to avoid rate limiting. Without a token, you'll be limited to 60 requests per hour.
+
+```bash
+export GITHUB_TOKEN=your_token_here
+uv run setup-test-data
+```
+
+## Output
+
+The script downloads actual binary release assets and saves release metadata:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)